### ISSUE TITLE
Windows large file support

### ISSUE THREAD

**Issue by bjeurissen** [MRtrix3/mrtrix3#1058]
Currently, large file support seems to be broken on 64-bit Windows.

Performing a fresh install from the current master branch, reading a file larger than 2GB results in a "file is smaller than expected" error (note that the FS is NTFS). The same file can be read without issues on Linux and Mac. 

After reducing the file size below 2GB by cropping the image or removing volumes, the reading the file becomes possible on Windows as well.

**Comment by bjeurissen** [comment]
The full error output is below:
```
mrstats: [INFO] opening image "largefile.mif"...
mrstats: [DEBUG] reading key/value file "largefile.mif"...
mrstats: [DEBUG] sanitising image information...
mrstats: [WARNING] transform matrix contains invalid entries - resetting to sane defaults
mrstats: [DEBUG] memory-mapping file "largefile.mif"...
mrstats: [DEBUG] image "largefile.mif" unloaded
mrstats: [ERROR] file "largefile.mif" is smaller than expected
```

**Comment by jdtournier** [comment]
closed with #1060
